<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Begin CVS Header </comment>
<comment>   $Source: /fs/turing/cvs/copasi_dev/copasi/UI/CQGlobalQuantitiesWidget.ui,v $ </comment>
<comment>   $Revision: 1.3 $ </comment>
<comment>   $Name: Build-33 $ </comment>
<comment>   $Author: aekamal $ </comment>
<comment>   $Date: 2010/01/18 15:53:07 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2008 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>CQGlobalQuantitiesWidget</class>
 <widget class="CopasiWidget" name="CQGlobalQuantitiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>456</height>
   </rect>
  </property>
  <property name="caption" stdset="0">
   <string>GlobalQuantities</string>
  </property>
  <layout class="QGridLayout">
   <item row="8" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <spacer name="horizontalSpacerLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnNew">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnDelete">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnClear">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Delete All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QTableView" name="mpTblGlobalQuantities">
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="3" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mpLEFilter"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasiWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mpTblGlobalQuantities</tabstop>
  <tabstop>mpBtnDelete</tabstop>
  <tabstop>mpBtnClear</tabstop>
 </tabstops>
 <includes>
  <include location="global">vector</include>
  <include location="local">copasi/UI/copasiWidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnClear</sender>
   <signal>pressed()</signal>
   <receiver>CQGlobalQuantitiesWidget</receiver>
   <slot>slotBtnClearClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnDelete</sender>
   <signal>pressed()</signal>
   <receiver>CQGlobalQuantitiesWidget</receiver>
   <slot>slotBtnDeleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpTblGlobalQuantities</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>CQGlobalQuantitiesWidget</receiver>
   <slot>slotDoubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
     <sender>mpBtnNew</sender>
     <signal>pressed()</signal>
     <receiver>CQGlobalQuantitiesWidget</receiver>
     <slot>slotBtnNewClicked()</slot>
     <hints>
       <hint type="sourcelabel">
         <x>189</x>
         <y>434</y>
       </hint>
       <hint type="destinationlabel">
         <x>270</x>
         <y>227</y>
       </hint>
     </hints>
   </connection>
 </connections>
</ui>
